docs/20_faq.md

```diff
@@ -5,25 +5,122 @@ sidebar_label: FAQ
 ---
 
 
-# FAQ
+# Frequently Asked Questions
 
 :::info
 
-Frequently asked questions about the ChainSafe gaming SDK.
+Frequently asked questions about ChainSafe Gaming's SDK, web3.unity.
 
 :::
 
-**Q**: I want to connect to other EVM compatible chains but it only lets me connect to Ethereum. How do I do that?
+## Common Questions
 
-**A**: You will need to edit the **network.js** file found within the WebTemplates folder. Edit the ChainId to the one for the blockchain you want to game to hit.
+### Is the SDK free to use?&#x20;
 
-**Q:**  I'm seeing the following error when I install the package. **The type or namespace name 'Newtonsoft' could not be found**
+Yes, the SDK is free to use.
 
-**A:** NewtonSoft is a dependency of the SDK and needs to be installed for the SDK to work. This could be found on the Unity Asset store [JSON.NET](https://assetstore.unity.com/packages/tools/input-management/json-net-for-unity-11347)
+### Can I make desktop games with the SDK?
 
-**Q:** I'm getting the following error when trying to import an NFT texture into the SDK. **Curl error 1: Protocol "ipfs" not supported or disabled in libcurl**
+Yes, you can make Desktop / Web and Mobile games using our SDK. You will need to create a Web3GL build to make desktop / mobile games with web3.unity.
```

Comment:
This is confusingly worded and also the capitalization is not consistent.

This also highlights a need to update the docs about what exactly both WebGL (and Web3GL) are since [this section](https://docs.gaming.chainsafe.io/webgl) is largely lacking adequate description that would aid this FAQ.

We will need to ascertain from the Gaming team what the status is of [our Web3GL](https://github.com/ChainSafe/game-web3gl) repo as well as the [accompanying Web3GL site](https://chainsafe.github.io/game-web3gl/) we have up for it.

<img width="1309" alt="Screen Shot 2023-02-05 at 8 09 14 PM" src="https://user-images.githubusercontent.com/17755587/216881818-06a8d1bc-9972-452c-b99f-3fecb76eb9c8.png">